spec.v2.yaml

```diff
@@ -90,41 +90,54 @@ data:
     # the runtime block so that it matches the stream the module was built
     # against.  Multiple builds result in multiple output modulemd files.
     # See below for an example.
-    # TODO: Provides, conflicts, obsoletes, recommends, etc.
-    #       Do we even need those?
+    # Provides also support stream expansion and indicate what modules names
+    # and streams are provided.  Just like with dependencies, streams can
+    # be inclusive or exclusive lists, with an empty list meaning that any
+    # stream of that module name is provided.
     # The example below illustrates how to build the same module in four
     # different ways, with varying build time and runtime dependencies.
     dependencies:
         # Build on all available platforms except for f27, f28 and epel7
         # After build, the runtime dependency will match the one used for
         # the build.
+        # In this situation this module also provides "onestream:1".
         - buildrequires:
               platform: [-f27, -f28, -epel7]
           requires:
               platform: [-f27, -f28, -epel7]
+          provides:
+              onestream: [v1]
```

Comment:
why do we need multiple keywords for same thing?

Reply:
What do you mean?